### PR TITLE
Dev/165 Average score should show "0" if there are not questions in the exam

### LINKED ISSUE #165: [Bug] Average score should show "0" if there are not questions in the exam
image

### PR DESCRIPTION
**Pull request by snowmancc**
resolve #165 

image

image

### CODE REVIEW

**Review by Chaoyuuu — APPROVED** [review]
LGTM! 😄

File: src/services/ExamTranscriptService.js [hunk 1]
```diff
@@ -35,13 +35,14 @@ class ExamTranscriptService extends AbstractService {
...
       const ids = examOverview.questions.map(question => question.problemId);
       await this.problemService.getProblemsByIds(ids)
         .then(problems => problems
           .forEach(problem => totalTestCasesOf[problem.id] = problem.testcases.length));
-      return {examinees, examOverview, examTranscript, totalTestCasesOf};
-    }).then(data => new ExamScoreboard({...data}));
-  }
-}
-
-export {ExamTranscriptService};
```
Comment (Wally5077) [inline src/services/ExamTranscriptService.js:47]: Change it back.

**Review by wallychen0705 — CHANGES_REQUESTED** [review]
Review done

**Review by wallychen0705 — APPROVED** [review]
LGTM

**Pull request merged**